app/static/tests/unit/components/sensitivity/sensitivitySummaryPlot.test.ts

```diff
@@ -77,7 +77,7 @@ describe("SensitivitySummaryPlot", () => {
                     namespaced: true,
                     state: {
                         batch: hasData ? mockBatch : null,
-                        plotSettings
+                        plotSettings: { ...plotSettings }
```

Comment:
Weird jest errors occurred by reusing old plot settings which had been observed by reactivity in previous tests.